### ISSUE TITLE
pd.read_excel() has some parameters incorrectly documented

### ISSUE THREAD

**Issue by Dr-Irv** [microsoft/pylance-release#385]
## Environment data

-   Language Server version: 2020.9.5
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): Anaconda 3.7.5

## Code Snippet / Additional information

The stubs for `read_excel` have some errors with respect to the `header` and `skiprows` arguments.  The following diffs in the file `dist/bundled-stubs/pandas/io/excel/_base.pyi` fix these issues:
```
diff _base.pyi.ORIG _base.pyi
10c10
<     header: Union[int, Sequence[int]] = ...,
---
>     header: Optional[Union[int, Sequence[int]]] = ...,
20c20
<     skiprows: Optional[Sequence[int]] = ...,
---
>     skiprows: Optional[Union[Sequence[int], int, Callable]] = ...,
271c271
<     header: Union[int, Sequence[int]] = ...,
---
>     header: Optional[Union[int, Sequence[int]]] = ...,
281c281
<     skiprows: Optional[Sequence[int]] = ...,
---
>     skiprows: Optional[Union[Sequence[int], int, Callable]] = ...,
```

### Details on why these changes are needed:

1. For `header`, the value `None` is allowed
2. For `skiprows`, the docs need to be updated.  See https://github.com/pandas-dev/pandas/issues/36435

**Comment by Dr-Irv** [comment]
Note - when I first reported this, I thought that `sheetname` was wrong, but now I see what the `@overload` is doing, so all is well.  Updated the original text above to remove that part.

**Comment by gramster** [comment]
For skiprows, this is motivation to start making PRs against pandas to inline these types or we'll be whacking moles endlessly.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.9.6, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202096-23-september-2020
